docusaurus/docs/guides/hsm/index.md

```diff
@@ -1,10 +1,8 @@
 ---
-sidebar_position: 3
 id: hsmIndex
+title: Hardware Security Modules

```

Comment:
PKCS11 is relevant here as it is an API to interact with HSMs. I don't think removing it is necessary

Reply:
Ah, I may have misunderstood the intent of the article. We're specifically documenting our support of PKCS#11, not HSMs in general, and not other HSM interfaces like PARSEC, right?

Reply:
The page references and uses pkcs11. i just don't feel the need to remove it from the title and i'm trying to reduce changes that aren't related to "running ziti in production". i'd like to keep changes like this outside of the scope of this pr